src/Umbraco.Forms.Integrations.Commerce.EMerchantPay/Helpers/MappingFieldHelper.cs

```diff
@@ -0,0 +1,48 @@
+﻿#if NETCOREAPP
+using Microsoft.Extensions.Options;
+#else
+using System.Configuration;
+#endif
+
+using System.Collections.Generic;
+using System.Linq;
+
+using Umbraco.Forms.Integrations.Commerce.EMerchantPay.Configuration;
+
+namespace Umbraco.Forms.Integrations.Commerce.EMerchantPay.Helpers
+{
+    public class MappingFieldHelper
+    {
+        private readonly PaymentProviderSettings _paymentProviderSettings;
+
+#if NETCOREAPP
+        public MappingFieldHelper(IOptions<PaymentProviderSettings> options)
+        {
+            _paymentProviderSettings = options.Value;
+        }
+#else
+        public MappingFieldHelper()
+        {
+            _paymentProviderSettings = new PaymentProviderSettings(ConfigurationManager.AppSettings);
+        }
+#endif
+
+        public IEnumerable<string> GetMappings()
+        {
+            if (string.IsNullOrEmpty(_paymentProviderSettings.MappingFields))
+                return Enumerable.Empty<string>();
+
+            try
```

Comment:
I think it would probably be better if this just threw an exception, as silently continuing may be harder to debug.

Or if you do want to continue on error, better to be explicit and do some checks that the string is in the expected format.